mitodl/odl-video-service
# Pre-commit checks

#### What are the relevant tickets?
N/A

#### What's this PR do?
- Installs a pre-commit hook during `docker-compose build` that will run before every commit

#### How should this be manually tested?
- run `pip install pre_commit detect-secrets`
- run `pre-commit install`
- `git add` an `.env`-like file with fake (but realistic) AWS credentials in it
- `git commit -a -m 'test' ` should generate an error list of bad things it finds in the added file, and the commit should be aborted.
- Replace the contents of the file with safe random text, then run `git commit -a -m 'test'` again, it should work.
